recipes/rmm/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "rmm" %}
+{% set version = "21.12.00" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rapidsai/rmm/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 71802422a04f2e0a828f7b7ec77e5cd1d24361d6dc5bf60a534659e4c2e08a44
+  folder: rmm
+
+build:
+  number: 0
+  skip: true  # [(not linux) or (cuda_compiler_version ==  "None") or (aarch64) or (py < 37) or (py >= 39)]
```

I've talked to the RMM team and I think the Python constraint can be relaxed. Also, we need to use CUDA 11.2 to build:
```suggestion
  skip: true  # [(not linux) or (cuda_compiler_version !=  "11.2") or (aarch64)]
```

---

Done.

---

Looks like upstream has two build variants: with/without cuda malloc async. Is it valuable to build without cuda malloc async?

---

We can do that but from my perspective it's not critical here.

---

@leofang 
The build is failing due to  "Package 'rmm' requires a different Python: 3.9.10 not in '<3.9,>=3.7'". So need to wait until RMM team does changes or for now I can back the same python constraints and can be removed once RMM team has done the changes.

---

@alhad-deshpande see my https://github.com/conda-forge/staged-recipes/pull/17986#issuecomment-1040591473 below. If you wanna go with the patch route, you can save the diff to a file (say, `py_ver.diff`), include it in the recipe folder, and [add a patch entry in the source section](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html?highlight=rpath#patches).

---

@leofang 
Build for py37 and 38 for now.